mtx/sidecar/package.json

```diff
@@ -24,13 +24,16 @@
     }
   },
   "devDependencies": {
-    "sqlite3": "^5"
+    "sqlite3": "^5",
+    "forever": "^4"
   },
   "engines": {
     "node": "^16"
   },
   "scripts": {
     "start": "cds run",
-    "build": "npm install && npx cds build ../.. --for mtx-sidecar && cd gen && npm install"
+    "build": "npm install && npx cds build ../.. --for mtx-sidecar && cd gen && npm install",
+    "startAsync": "forever start -o out.log -e err.log -c node node_modules/@sap/cds/bin/cds.js run --profile development",
```

Comment:
would be nice if we could somehow execute `cds run` or `cds watch`; not sure if this is possible

@mofterdinger - what do you think?

Reply:
This https://pm2.keymetrics.io/docs/usage/quick-start/ could be a candidate. Not sure if it can just daemonize command.

Reply:
The MTXS sidecar needs to be started in the background, otherwise the maven build is blocked at the steps, where the sidecar is started. The [forever npm module](https://www.npmjs.com/package/forever) allows to start a node application asynchronously in background and fits perfect to our needs. But I'm not sure if we want to demonstrate this setup in our sample bookshop, but this is other kind of question.

Reply:
@beckermarc requested it.

Reply:
ok, thanks, I wasn't aware of that request. Then I'm fine with it.